### DISCUSSION TITLE
[deepin-gomoku] It would be better to complete gomoku manager for loading AI engines.

### DISCUSSION THREAD

**Discussion by Joker2770** [linuxdeepin/developer-center#3450] [Features Request & Ideas | 特性请求 & 头脑风暴]
There is a good program of the gomoku manager to load and match with AI engine. But it's a MFC program only working on Windows system.
This program named [piskvork](https://sourceforge.net/projects/piskvork/), and can be found at [download page of gomocup manager](https://gomocup.org/download-gomocup-manager/).
The manager communicating with AI engines under the text protocol as follow.
[gomocup protocol](http://petr.lastovicka.sweb.cz/protocl2en.htm)

That I sincerely hope deepin-gomoku could be a manager like piskvork.

**Comment by BLumia** [comment]
cc @meiyixiang
